Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - Multitenant Isolation.postman_collection.json

```diff
@@ -42,12 +42,8 @@
 											"            pm.collectionVariables.set(\"CreatedOdsInstanceId\", id);\r",
 											"        }\r",
 											"    });\r",
-											"} else {\r",
-											"    //stop running tests\r",
-											"    postman.setNextRequest(null);\r",
```

Comment:
This Postman collection previously used a postman function `postman.setNextRequest(null);` to short circuit out of the collection when the environment is not set for multitenant mode. When we run the job in our github actions, the collections are combined into one large collection, and this collection ends up somewhere in the middle of this new large collection. As a result, when the environment is run in single-tenant mode and that function is called, it short circuits all of the rest of the combined collection, causing much of the testing to be skipped in single-tenant mode. This fixes that by removing the `postman.setNextRequest(null);` and instead checking for multitenant mode before executing any and all tests in this particular collection.